src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
 public class SalaryInfo {
+    private static final Integer HOURS_OF_WORK_INDEX = 2;
+    private static final Integer SALARY_PER_HOUR_INDEX = 3;
+    private static final Integer POSITION_OF_DATE_IN_STRING = 0;
+    private static final Integer START_DAY_INDEX = 0;
+    private static final Integer START_MONTH_INDEX = 2;
+    private static final Integer END_DATE_INDEX = 3;
+    private static final Integer END_MONTH_INDEX = 5;
+    private static final Integer DAYS_IN_MONTH = 30;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int startDay = calculateDays(dateFrom);
+        int endDay = calculateDays(dateTo);
```

Comment:
this how you can use `LocalDateTime` class and compare it rather than using `int`